TELCODOCS-2064: Docs and RN: CNF-14082 SR-IOV Operator support for Intel C741 Emmitsburg Chipset (#84760)
openshift/openshift-docs: modules/nw-sriov-configuring-operator.adoc

|`boolean`
|Specifies whether to enable or disable the SR-IOV Network Operator metrics. By default, this field is set to `false`.

> |`spec.featureGates.mellanoxFirmwareReset`

Reviewer: This is a Tech Preview feature. Can we have a TP note here somehow?

Author: Thanks @cgoncalves for your feedback. I have included the standard Technology Preview text snippet that appears with all Technology Preview features.

|`boolean`

> |Specifies whether to reset the firmware in the SR-IOV Network Operator. By default, this field is set to `true`.

Reviewer: Feel free to re-phrase my suggestion:

```suggestion
|Specifies whether to reset the firmware on VF changes in the SR-IOV Network Operator. Certain chipsets (e.g., Intel C740 Series) do not completely power off the PCI-E devices, which is required to configure VFs on NVIDIA/Mellanox NICs. By default, this field is set to `false`.
```

Author: Thanks @cgoncalves for your feedback. I have included, and slightly re-phrased, your suggestion.

|`boolean`
|Specifies whether to enable or disable the SR-IOV Network Operator metrics. By default, this field is set to `false`.

|`spec.featureGates.mellanoxFirmwareReset`
|`boolean`

> |Specifies whether to reset the firmware on virtual function (VF) changes in the SR-IOV Network Operator. Some chipsets (for example, Intel C740 Series) do not completely power off the PCI-E devices, which is required to configure VFs on NVIDIA/Mellanox NICs. By default, this field is set to `false`.

Reviewer 2: ```suggestion
|Specifies whether to reset the firmware on virtual function (VF) changes in the SR-IOV Network Operator. Some chipsets, such as the Intel C740 Series, do not completely power off the PCI-E devices, which is required to configure VFs on NVIDIA/Mellanox NICs. By default, this field is set to `false`.
```
Avoiding parenthesis: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses
Using "such as" instead of "for example": https://www.ibm.com/docs/en/ibm-style?topic=format-examples
